openapi/openapi-collections.ytt.yaml

```diff
@@ -200,4 +200,26 @@ paths:
           required: true
           schema:
             type: string
-      responses: #@ response(reference("CollectionClusterInfo"))
\ No newline at end of file
+      responses: #@ response(reference("CollectionClusterInfo"))
+
+    post:
+      tags:
+        - collections
+        - cluster
+      summary: Update collection cluster setup
+      operationId: update_collection_cluster
+      requestBody:
+        description: Collection cluster update operations
+        content:
+          application/json:
+            schema:
+              $ref: "#/components/schemas/ClusterOperations"
+      parameters:
```

Comment:
it also requires definition for `{collection_name}` in `path`

Reply:
good catch :+1: 
https://github.com/qdrant/qdrant/pull/869/commits/c05f7a74840febc7e0b610a92e56af1f08bd3e97